### DISCUSSION TITLE
Wish: GNU/Linux integration

### DISCUSSION THREAD

**Discussion by rzr** [neutralinojs/neutralinojs#1174] [Ideas]
Hi,

I am wondering if this software is already packaged in some GNU/Linux distros.
If yes this would be nice to add link to readme page (and website), 
If not I propose this meta ticket to track packaging effort.

Regards

**Comment by shalithasuranga** [comment]
Hey.. thanks for posting this :). Could you please let us know whether you mean Neutralinojs system dependencies (Eg: GTK, WebKit GTK), etc are not packaged in some distros?

**Comment by rzr** [comment]
Deps might be there , but not Neutralinojs itself, it could be integrated to make it more usable to distro's users.
Myself I have interest for Debian, and OpenEmbedded/Yocto (for Oniro project) to say a few.
